<?xml version="1.0" encoding="ISO-8859-15"?>
<launch>
	<arg name="slam_name" value="/picoflexx/cam0"/>
	
	<!-- Run Bagfile -->
     <node pkg="rosbag" type="play" name="player" output="screen"   
      args="-r 1.0 /home/karrer/FinalDataSet/2016_Feb_12/Picoflexx/Desk/Mode1/desk_mode1_1.bag --clock"/> 
      <!--/home/karrer/FinalDataSet/2016_Feb_11/Picoflexx/Boiler/boiler_mode1_oneside2.bag" -->
      
	<!-- Run rovio -->
     <!--node  name="rovio" pkg="rovio" type="rovio_node"     
     clear_params="true" output="screen"--> 
     <!--/node-->


	<!-- Run fastfusion_node-->

	<node  name="fastfusion_node" pkg="fastfusion_node" type="fastfusion_node" clear_params="true" output="screen" >
		<remap from="image_depth"		to="$(arg slam_name)/depth/image_raw" />
		<remap from="image_conf"		to="$(arg slam_name)/confidence/image_raw" />
		<remap from="image_noise" 		to="$(arg slam_name)/noise/image_raw" />
		<remap from="rovio_odometry"	to="/rovio/odometry" />
		<remap from="point_cloud" 		to="/camera/depth/points" />

	<!-- Load parameters -->
 	<rosparam file="$(find fastfusion_node)/cfg/params.yaml"/> 
 	<rosparam file="/home/karrer/FinalDataSet/2016_Feb_12/Picoflexx/Rig_Calibration/run2/calibration_params.yaml"/>
 	<!--rosparam file="$(find fastfusion_node)/cfg/camchain_realsense.yaml"-->
	</node>
	
	
	<!--node-->
</launch>
